front/src/reducers/main.ts

```diff
@@ -5,6 +5,17 @@ import { ThunkAction, Notification } from '../types';
 //
 // Actions
 // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+export const UPDATE_LANGUAGE = 'main/UPDATE_LANGUAGE';
+type ActionUpdateLangage = { type: typeof UPDATE_LANGUAGE };
+export function updateLangage(language: string): ThunkAction<ActionUpdateLangage> {
+  return (dispatch) => {
+    dispatch({
+      type: UPDATE_LANGUAGE,
+      language,
+    });
+  };
+}
```

Comment:
```suggestion
```
This is not used, so we should not commit this (dead code), same for all the changes of this file

Reply:
Oh, it was a first try, never used finally (i18n has it's own localstorage variable!). Thanx !